List_Indexing.py

```diff
@@ -0,0 +1,21 @@
+##1. List Indexing - take the same problem done for strings, but this time for a list. List can be either hardcoded in the program itself, like:
```

Comment:
Program not working.

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/38807024/203931503-5b895454-7871-4c8d-93f5-f3111f03d32e.png">